### PR TITLE
Add Persian (Farsi) FastConformer Hybrid Large variant

### PR DESCRIPTION
**Pull request by Evilenor**
## Why

Handy currently has no lightweight, CPU-only option for Persian (Farsi). The existing choices each have a real gap:

- **Whisper (small/medium)**: officially support Persian, but accuracy is noticeably weaker — Persian is a low-resource language in Whisper's training mix.
- **Whisper large-v3**: the first Whisper size with genuinely usable Persian accuracy, but it's GPU-bound. On a shared or modest GPU, model load and inference compete with everything else using the GPU, making it slow and inconsistent in practice — the opposite of the instant, CPU-only feel Canary/Parakeet give for their languages.
- **Other lightweight community options** (Vosk, wav2vec2-persian): benchmarked WER around 45-47% on real-world/noisy Persian speech (PSRB benchmark), i.e. not meaningfully better than what's here, with none of the Handy integration.

NVIDIA already publishes a FastConformer checkpoint for Persian (`nvidia/stt_fa_fastconformer_hybrid_large`, 115M, Common Voice fa-15.0 fine-tune of the English FastConformer encoder) that had never been converted for transcribe.cpp. It fills exactly this gap: same architecture family as Parakeet, so once converted it runs with the same CPU-only, instant-load, no-GPU-contention profile as Canary/Parakeet.

**Accuracy, honestly**: this is a Common Voice-only fine-tune, so — like most Common-Voice-trained models — it's strong on clean/formal audio and weaker on noisy or spontaneous speech. It is not a universal replacement for Whisper large-v3's accuracy ceiling on hard audio. It's the option for the (common) case where GPU contention/latency matters more than squeezing out the last few points of WER on noisy recordings — e.g. dictation in a normal room. In initial testing, in a quiet environment, it's been consistently accurate at Canary-like speed.

## What changed in the code

The checkpoint isn't in the known-slug list, and its architecture is a 'reverse hybrid': the primary decoder/joint is RNNT, and the CTC head we want to walk (`head_kind="ctc"`) lives under the auxiliary `ctc_decoder.` prefix instead of top-level `decoder.` — the opposite of `parakeet-tdt_ctc-*`, which drops `ctc_decoder.` the other way around. The existing CTC tensor-mapping table assumes the CTC decoder lives at `decoder.decoder_layers.0.*` (true only for the pure `parakeet-ctc-*` checkpoints, which have no RNNT head at all). Three changes to `convert-parakeet.py`, all additive:

1. **New `VARIANT_PROFILES` entry** (`stt-fa-fastconformer-hybrid-large`): registers the model's display name, size (115M), vocab_size (1024), license (cc-by-4.0, matching NVIDIA's original), and language (`fa`), following the exact same shape as the existing entries.

2. **New opt-in `ctc_from_aux` profile flag.** Right after `sd = model.state_dict()` loads, if this flag is set on the profile, the code aliases every `ctc_decoder.*` key onto a matching `decoder.*` key in the state dict:
```python
if profile.get("ctc_from_aux") and head_kind == "ctc":
    for k in list(sd.keys()):
        if k.startswith("ctc_decoder."):
            sd["decoder." + k[len("ctc_decoder."):]] = sd[k]
```
This makes the existing CTC tensor-mapping table's fixed `decoder.decoder_layers.0.*` lookups resolve to the right tensors, with zero changes to that table or to any other variant's conversion path — the flag defaults to unset/False everywhere else.

3. **Small extension to `is_expected_unused()`**: when `ctc_from_aux` is set, the now-genuinely-unused RNNT `decoder.prediction.*` / `joint.*` tensors, and the original (now aliased-away) `ctc_decoder.*` keys, are recognized as expected-unused instead of printing as spurious unconsumed-tensor warnings.

Net effect: **no existing variant's behavior changes** — every change is gated behind the new flag, which only this one profile sets.

## The converted model

Converts cleanly to a 417MB F32 GGUF (677 tensors). I've hosted it here so anyone can try it without running the conversion themselves:

https://huggingface.co/evilenor/stt-fa-fastconformer-hybrid-large-gguf

Loaded into Handy's local custom-models folder, it runs with the same CPU-only, instant-load profile as Canary/Parakeet.

**Request**: would you be open to adopting this hosted GGUF as the Persian option in Handy — either linking to it directly, or mirroring/re-uploading it under the `handy-computer` org so it's selectable from the in-app model list the same way Canary/Parakeet are? Happy to transfer it, rename it, add a quantized variant, or make any other changes needed to fit how the other models are packaged — let me know what you'd prefer.

### CODE REVIEW

**Comment by cjpais** [comment]
Hey thanks for this and yes, I will get around to it. I will definitely host on handy-computer, but I will need to do the WER testing and things before doing it there directly

In the future in Handy we might allow people to just give a hf repo url or something and allow those in as well, but I need to clean things up over there still. There's a lot of bugs that need fixing so I'm trying to knock those down

**Comment by Evilenor** [comment]
@cjpais that is great ,thanks for Being so responsive

**Comment by cjpais** [comment]
I do apologize for not getting this in. I have just been quite busy. It will come in at some point specifically. Just don't quite have time yet. I have some bigger items that I'm really focused on, and I don't want to divert too much of my attention away at the moment

**Comment by Evilenor** [comment]
@cjpais Keep up the good work
